libpreferencepanes: new package

qmfrederik opened the pull request:
Packages the https://github.com/gnustep/apps-systempreferences library.

This package contains the library component only; and excludes the actual system preferences application and the supporting bundles.

The libpreferencepanes names was chosen in analogy with how this library is packaged in Ubuntu: https://packages.ubuntu.com/noble/libpreferencepanes1

ognevny commented on mingw-w64-libpreferencepanes/PKGBUILD:
```diff
@@ -0,0 +1,48 @@
+  make
+}
+
+check() {
```
useless `'check()`

ognevny commented on mingw-w64-libpreferencepanes/PKGBUILD:
```diff
@@ -0,0 +1,48 @@
+
+  make install DESTDIR="${pkgdir}"
+  rm -rf ${pkgdir}/${MINGW_PREFIX}/lib/GNUstep/Bundles/
+  rm -rf ${pkgdir}/${MINGW_PREFIX}/lib/GNUstep/Applications/
```
remove a `/` before ${MINGW_PREFIX}`

qmfrederik commented:
Thanks @ognevny , I pushed a commit which hopefully addresses your feedback

ognevny commented:
> Thanks @ognevny , I pushed a commit which hopefully addresses your feedback

I have one more :)

qmfrederik commented:
> I have one more :)

That's a real cliffhanger, are you going to let me know what it is? 😂

ognevny commented:
> > I have one more :)
> 
> That's a real cliffhanger, are you going to let me know what it is? 😂

I'm very, but wdym?

qmfrederik commented:
I believe your comments were that (i) there was a useless `check()`, which I removed, and (ii) there shouldn't be a trailing slash before `${MINGW_PREFIX}`, which I removed.

Is there anything else you wanted me to change in this PR?

ognevny commented:
> I believe your comments were that (i) there was a useless `check()`, which I removed, and (ii) there shouldn't be a trailing slash before `${MINGW_PREFIX}`, which I removed.
> 
> Is there anything else you wanted me to change in this PR?

uh oh, GitHub mobile moment. I also put a review about useless `prepare()`

qmfrederik commented:
Allrighty, good point. Fixed that ;-)

lazka commented on mingw-w64-libpreferencepanes/PKGBUILD:
```diff
@@ -0,0 +1,40 @@
+license=('spdx:LGPL-2.1-or-later')
+makedepends=("${MINGW_PACKAGE_PREFIX}-autotools"
+             "${MINGW_PACKAGE_PREFIX}-clang"
+             "${MINGW_PACKAGE_PREFIX}-lld"
```
Just wondering, is ldd actually used here?

qmfrederik replied:
Yes, the GNUstep build system picks up the configuration which were used when building gnustep-make.

For example, if you run `gnustep-config --variable=LDFLAGS`, it will echo back the `LDFLAGS` which are defined in [mingw-w64-gnustep-make/PKGBUILD](https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-gnustep-make/PKGBUILD#L28):

```
$ gnustep-config --variable=LDFLAGS
-fuse-ld=lld -lstdc++ -lgcc_s
```

and that's how they eventually end up being used in the build process for this package.

lazka replied:
Aha! thanks for the explanation
